examples/translations/scripts/lib/printer.js

```diff
@@ -23,18 +23,20 @@ export default function printICUMessage(ast) {
 
         let formatType = format.type.replace(/Format$/, '');
 
+        let style, offset, options;
```

Comment:
Otherwise can remove this rule:

`29:9  error  Unexpected lexical declaration in case block  no-case-declarations`